module/PowerNSX.psm1

```diff
@@ -26657,6 +26657,77 @@ function Add-NsxServiceGroupMember {
     end {}
 }
 
+function Remove-NsxServiceGroupMember {
+
+    <#
+    .SYNOPSIS
+    Removes a single Service, numerous Services, or a Service Group to a Service
+    Group
+    
+    .DESCRIPTION
+    Removes the defined Service or Service Group to an NSX Service Groups. Service
+    groups contain a mixture of selected ports to represent a potential
+    grouping of like ports.
+    
+    This cmdlet removes the defined Services or Service Groups within a Service
+    Group for specific or all Service Groups 
+    
+    .EXAMPLE
+    Get-NsxServiceGroup Heartbeat | Remove-NsxServiceGroupMember -Member $Service1
+    
+    Remove all saved Service Group Members named Heartbeat in Service Group in $Service1
+    
+    Get-NsxServiceGroup Service-Group-4 | Remove-NsxServiceGroupMember $Service1,$Service2 -Confirm:$false
+    
+    Remove all saved Service Group Members named Heartbeat in Service Group in $Service1 and $Service2 without prompting for confirmation
```

Comment:
again the comment is not so clear.  There is no mention of heartbeat in the cli example?